packages/superset-ui-chart-controls/src/shared-controls/components.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import RadioButtonControl from '../components/RadioButtonControl';
+
+export * from '../components/RadioButtonControl';
+
+/**
+ * Aliases for Control Components
+ */
+export default {
+  RadioButtonControl,
+};
```

Comment:
The intention is we should slowly move all control components in [superset-frontend/src/explore/components/controls/index.js](https://github.com/apache/incubator-superset/blob/291306392443a5a0d0e2ee0cc4a95d37c56d4589/superset-frontend/src/explore/components/controls/index.js#L44) here.

Currently the API is compatible to have them in both places.